xcodeproj/internal/templates/legacy_installer.sh

```diff
@@ -134,7 +134,7 @@ rsync \
   --chmod=u+w,F-x \
   --exclude=project.xcworkspace \
   --exclude=rules_xcodeproj/bazel \
-  --exclude=xcuserdata \
+  --exclude="*.xcbkptlist" \
```

Comment:
User created schemes should be left alone as well. Probably better to leave this exclude as is, and copy over the schememanagement file, similar to the incremental installer?

https://github.com/MobileNativeFoundation/rules_xcodeproj/blob/4f6860856f808e2e044637380daadc2d2090bc05/xcodeproj/internal/templates/incremental_installer.sh#L138-L142

Reply:
Sounds good, let me update this PR

Reply:
@yongjincho92 Any progress on this?